skype/skype-ps/skype/New-CsTenantDialPlan.md

```diff
@@ -110,7 +110,7 @@ The ExternalAccessPrefix parameter is a number (or set of numbers) that designat
 (For example, to tenant-dial an outside line, first press 9.) This prefix is ignored by the normalization rules, although these rules are applied to the remainder of the number.
 
 The OptimizeDeviceDialing parameter must be set to True for this value to take effect.
-This parameter must match the regular expression \[0-9\]{1,4}: that is, it must be a value 0 through 9 and one to four digits in length.
+The value of this parameter must be no longer than 4 characters long and can contain only digits, "#" or a "*".
 The default value is 9.
```

Comment:
Description for "-ExternalAccessPrefix" parameter says "The default value is 9", but it does not appear to be correct

Reply:
Correct. Checked.